index.html

```diff
@@ -1222,19 +1222,36 @@ <h1>Common Deployment Patterns</h1>
     </p>
 
     <section id="telemetry">
-      <h3>Telemetry</h3>    
-      <section class="ednote" title="TODO">
-      <h3>TODO</h3>
-      Create a description and a diagram for a telemetry scenario.
-      </section>
+      <h3>Telemetry</h3>  
+      <p>
+        Telemetry is the monitoring of remove devices, which implies automatic transmission to, and processing
+        of measurements and other data on a <a>Consumer</a>. 
+        Data may be transported over various bearers, both wireless and wired.
+        Examples include GSM networks, Bluetooth, WiFi, Ethernet, and other wired standards.
+      </p>
+      <p>
+        Remote metering devices include one or multiple sensors, in some cases they also contain an actuator.
+        In many cases the sensor data is transmitted at regular intervals, on a state change, 
+        or as a response to a request by the <a>Consumer</a>.
+        Remote metering devices are frequently small embedded systems with very limited resources. 
+        These may be battery powered and have to minimize power consumption through various measures,
+        e.g. sleep modes.
+      </p>
+      <p>
+        Typical user scenarios are monitoring of a high number of geographically distributed assets, 
+        examples include smart cities, factories, fleets, environmental monitoring and health monitoring. 
+        A common pattern is the one-to-many relationship between a single <a>Consumer</a> and a large number of devices.
```

Comment:
Even though I agree that this is one typical scenario, telemetry is not only this right? Usual people monitoring the energy usage of their home is also a typical scenario and there is no high number of geographically distributed assets.

Reply:
I think it is also important to point out that there is a relationship to the definition of _WoT events_ that can be associated with telemetry based scenarios.

Reply:
@egekorkan The home energy monitoring I would have seen in the "Device Controllers" section.

Reply:
@egekorkan, @sebastiankb Thanks for your feedback.
I have updated the section to address your comments.